README.md

```diff
@@ -96,6 +96,50 @@ color: forestGreen (#228B22)
 
 `Work in Progress`: used for pull requests that are still not ready to be merged but are created because you need help or in-between review.
 
+## resmio flow ##  
+>
+```
+resmio flow, bugs arrive like butterflies
+Oh, we don't know, so we chase them away
+Someday yet, we'll begin our sprint again
+Sprint again, sprint again...
+```
+ Pearl Jam - resmio flow
+
+## Team Timeline
+
+- **Thursday**: Review of issues still open / Estimation meeting
+- **Friday**: Sprint is compiled
+- **Monday**: Old Sprint ends / New Sprint starts ...
+
+## Personal Timeline
+
+### Which issue should you work on?
+1. If you have an opened Pull Request with required changes work on that first.
+2. Check if the are opened Pull Requests from your domain that need to be code reviewed and do that.
```

Comment:
Yes, maybe we can change the sentence, but the logic is the same. 

`Check if there are any open Pull Request where you are assigned on to be code or product reviewed.`

and maybe an extra one


`Check if there are any open Pull Request where you are assigned on work on that` – something like this. For example, when someone asks for help or review, or if some of your work is missing, etc.